HealthcareApp/src/MigrationAdminPanel/Configuration/MyApp.cs

```diff
@@ -0,0 +1,25 @@
+using MigrationAdminPanel.Services;
+
+namespace MigrationAdminPanel.Configuration
+{
+    public class MyApp
+    {
+        private readonly JsonMigrationsService _jsonMigrator;
+        private readonly XmlMigrationsService _xmlMigrator;
+
+        private const string JsonRelativePath  = @"/Users/vladmakarenko/Desktop/Dotnet Mentorship/HealthcareApp/src/MigrationAdminPanel/Data/sampleData.json";
+        private const string XmlRelativePath = @"/Users/vladmakarenko/Desktop/Dotnet Mentorship/HealthcareApp/src/MigrationAdminPanel/Data/sampleData.xml";
```

Comment:
Make parametrized because migration will be done on a remote machine but not on your PC probably